python/idsse_common/idsse/common/utils.py

```diff
@@ -206,3 +209,43 @@ def round_half_away(number: float, precision: int = 0) -> Union[int, float]:
         else _round_away_from_zero(factored_number)
     ) / factor
     return int(rounded_number) if precision == 0 else float(rounded_number)
+
+def shrink_grib(filename: str, variables: list[str]) -> None:
```

Comment:
This should be ', variable: List[str]' where List is imported from typing. Once we get mypy added to the check in process this will be flagged.

Reply:
Updated